# 3.6 Handling Exceptions

# Example 

```dart
void main(){
  String name = "Dart";

  try{
    print ('Name: $name');
    // The following line generates a RangeError
    name.indexOf(name[0], name.length - (name.length+2));
  } catch (exception) {
    print ('Exception: $exception');
  } finally {
    print ('Mission completed!');
  }
}

```
